src/launch/resources/pack.mcmeta

```diff
@@ -0,0 +1,6 @@
+{
+    "pack": {
+        "description": "baritoe",
```

Comment:
should be `baritone`

Reply:
also is this required by forge type poz?

Reply:
not required but you get a warning screen

Reply:
baritoe is correct